README.md

```diff
@@ -385,6 +386,53 @@ config := hazelcast.NewConfig()
 config.NetworkConfig().AddAddress("some-ip-address:port")
 hazelcast.NewClientWithConfig(config)
 ```
+
+### 1.4.3. Hazelcast Properties
+
+Hazelcast property is a property which can be set programmatically through `config.SetProperty` or with an environment variable.
+The value of this property will be:
+
+* the programmatically configured value, if programmatically set,
+* the environment variable value, if the environment variable is set,
+* the default value, if none of above is set.
+
+For example: 
+
+```go
+InvocationTimeoutSeconds = NewHazelcastPropertyInt64WithTimeUnit("hazelcast.client.invocation.timeout.seconds",
+    120, time.Second)
+```
+
+
+as we see `invocation timeout` is `120 seconds` by default. You can change this as follows:
+
+#### Programmatically:
+
+```go
+config.SetProperty(property.InvocationTimeoutSeconds.Name(), "2") // Sets invocation timeout as 2 seconds
+```
+
+or 
+
+```go
+config.SetProperty("hazelcast.client.invocation.timeout.seconds", "2") // Sets invocation timeout as 2 seconds
+```
+
+#### Via an Environment Variable:
+
+```go
+os.Setenv(property.InvocationTimeoutSeconds.Name(), "2")
+```
+
+
+If you set a property both programmatically and via an environment variable, the programmatically
+set value will be used.
+
+See [Client Properties](https://github.com/hazelcast/hazelcast-go-client/blob/master/config/property/client_properties.go) for the complete list.
```

Comment:
See the [complete list](https://github.com/hazelcast/hazelcast-go-client/blob/master/config/property/client_properties.go) of client system properties, along with their descriptions, which can be used to configure your Hazelcast Go client.